test/e2e/BrsComponents.test.js

```diff
@@ -526,4 +526,21 @@ describe("end to end brightscript functions", () => {
             "5",
         ]);
     });
+
+    test("components/scripts/FieldChangeRunner.brs", async () => {
+        outputStreams.root = __dirname + "/resources";
```

Comment:
Let's be sure to reset `outputStreams.root` — looks like it only gets set up once at the beginning of this suite!

https://github.com/sjbarag/brs/blob/5e843c121d17c6f982876762a10b9f9176860960/test/e2e/BrsComponents.test.js#L9-L21

Reply:
It turns out that all of these tests succeed because `root` was set in one of them, then never cleaned up (so the change propagated to all of them, allowing them to succeed). Because of that, and since these are all component tests, I just assigned `outputStreams.root` in the `beforeEach` for the entire suite.